contracts/NestedFactory.sol

```diff
@@ -65,6 +70,7 @@ contract NestedFactory is INestedFactory, ReentrancyGuard, OwnableProxyDelegatio
         reserve = _reserve;
         feeSplitter = _feeSplitter;
         weth = _weth;
+        withdrawer = new Withdrawer(_weth);
```

Comment:
🔧 Should we consider deploying the withdrawer on its own, and reuse it across the different factories, instead of deploying one every time?

Reply:
yes I agree

Reply:
Fixed ✅